packages/base/any/iproute2/builds/0003-police-Add-support-for-json-output.patch

```diff
@@ -0,0 +1,134 @@
+From 93de5dcc58c25da615cb72c015302f6c240e1bd2 Mon Sep 17 00:00:00 2001
+From: Roi Dayan <roid@nvidia.com>
+Date: Mon, 7 Jun 2021 09:44:08 +0300
+Subject: [PATCH] police: Add support for json output
+
```

Comment:
Please mention [upstream commit hash](https://git.kernel.org/pub/scm/network/iproute2/iproute2-next.git/commit/tc/m_police.c?id=0d5cf51e0d6c7bfdc51754381b85367b5f8e254a) in here.